### ISSUE TITLE
New message/order/customer notification bubble not updated live

### ISSUE THREAD

**Issue by costads** [PrestaShop/PrestaShop#23416]
#### Describe the bug

New message/order/customer notification bubble is not updated live, without refreshing the browser

#### Expected behavior

I expected a notification bubble number on either a new message/customer/order without refreshing the browser like in 1.7.6 and previous years versions.

#### Steps to Reproduce

Open frontpage of BO, and notice the notification bubbles does not update the numbers as you get new orders/messages.

**Screenshots**

If applicable, add screenshots or screen recordings to help explain your problem.

#### Additional information

* PrestaShop version: 1.7.7.1
* PHP version: 7.3

**Comment by florine2623** [comment]
Hello @costads ,
Even in previous version PS 1.7.6.9, the notification did not update without a refresh, as you can see on my screen record :
https://drive.google.com/file/d/1PdTdd4O63EiUNVbFbOsBySTrzG4PAIb9/view

Thanks!

**Comment by hibatallahAouadni** [comment]
Related to #9550

**Comment by costads** [comment]
Well, it does that on both my domains running 1.7.6.1. I just created a new user, and sent a message, and after a couple of minutes the bubbles appeared, both in BO, and at the tab in the browser without me touching Chrome at all. It has been like that for as long as i remember.
It was the very first thing I noticed in 1.7.7.1 that was annoying.

This is a screenshot taken after a few minutes, without any refresh.

![nitify](https://user-images.githubusercontent.com/67175129/109360476-98898700-7887-11eb-942e-e14667760575.jpg)
